### PR TITLE
ENH: Add NeuroNet paper review.

### PR DESCRIPTION
**Pull request by jhlegarreta**
Add NeuroNet paper review.

### CODE REVIEW

File: deep-learning/_posts/2018-11-30-NeuroNet.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    venue:   "MIDL 2018"
+pdf: "https://openreview.net/pdf?id=Hks1TRisM"
+---
+
+
+# Highlights
+
+- NeuroNet is a deep convolutional neural network mimicking multiple popular
+and state-of-the-art brain segmentation tools including FSL[^FSL], SPM[^SPM],
+and MALPEM[^MALPEM].
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply deep-learning/_posts/2018-11-30-NeuroNet.md]: Addressed in fe5dbbf.

File: deep-learning/_posts/2018-11-30-NeuroNet.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+- An autoencoder is used based on a ResNet encoder and multiple FCN decoders,
+as many as the number $$k$$ of output segmentations coming from different
+tools.
+- Multitask loss: average categorical cross-entropy.
+- Each data is automatically segmented using three different state-of-the-art
+tools (FSL, SPM and MALP-EM) generating five outputs in total. The
+automatically generated label maps from the three tools serve as training data
+for NeuroNet.
+- For structures that are present in the output of all three tools such as
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply deep-learning/_posts/2018-11-30-NeuroNet.md]: May be the wording is not the best: on the one hand, the labels they use are not only the structures that are common to the three tools. When the label at issue (e.g. a given brain tissue) is available in the output of all three software packages, they use all of them as if they were separate inputs for the training. Then they compare "tool-wise". Does this answer the question?

File: deep-learning/_posts/2018-11-30-NeuroNet.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+# Discussion
+
+- Trained a single model from these complementary and partially overlapping
+label maps.
+- Powerful "all-in-one", multi-output segmentation tool.
+- Reduce inference time with respect to the state-of-the-art tools.
+
+
+# Conclusions
```
[the remark this answers is not in the document]
Comment (jhlegarreta) [reply deep-learning/_posts/2018-11-30-NeuroNet.md]: I grant that it does not seem the best approach. May be *Conclusions* should be renamed to *Final remarks* or *Final comments*, and the *Discussion* section be renamed to *Conclusions*. Does this look a better solution?

File: deep-learning/_posts/2018-11-30-NeuroNet.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (lemairecarl) [reply deep-learning/_posts/2018-11-30-NeuroNet.md]: It does clarify.

File: deep-learning/_posts/2018-11-30-NeuroNet.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (lemairecarl) [reply deep-learning/_posts/2018-11-30-NeuroNet.md]: I'm only judging by the sections I have seen in papers. A "Discussion" section talks about the downsides of the method, and the "Conclusion" section make a final summary.

**Pull request merged**